### PR TITLE
feat(configurations): digital_receipt

### PR DESCRIPTION
**Pull request by alghanor**
add enabled flag

https://tillhub.atlassian.net/browse/TM-6267

### CODE REVIEW

File: lib/v1/configurations/items/receipts.js [hunk 1]
```diff
@@ -180,12 +180,23 @@ module.exports = oneOf({
   type: 'object',
   additionalProperties: false,
   properties: {
+    digital_receipt: oneOf({
+      type: 'object',
+      additionalProperties: false,
+      properties: {
+        enabled: {
+          description: 'States if order receipts should be used',
```
Comment (Willgeorges) [inline lib/v1/configurations/items/receipts.js:188]: I think this is not about order receipts

Comment (alghanor) [reply lib/v1/configurations/items/receipts.js:188]: just fixed the typo, sorry

**Comment by Willgeorges** [comment]
LGTM

**Pull request merged**